org-configurations/default-configuration.yml

```diff
@@ -0,0 +1,88 @@
+---
+name: "default name"
+slug: "default-slug"
+
+# configuration variables for the server app
+server:
+  host: "default-host"
+  proxy_urls:
+    password_change: "password_change_url"
+    password_reset: "password_reset_url"
+    password_reset_confirm: "password_reset_confirm_url"
+    registration: "registration_url"
+    user_auth_token: "user_auth_token_url"
+
+# configuration variables for the client
+client:
+  languages:
+    - "en"
+    - "it"
+  default_language: "en"
+  components:
+    header:
+      logo:
+        url: null # logo url
+        alternate_text: null
+      links:
+        - text:
+            en: "link1" #link text in english
```

Comment:
I've tried to make most of the things configurable in frontend. Orgs can provide texts in different languages like this.

Reply:
:+1: